README.md

```diff
@@ -300,124 +247,65 @@ with the standard being `:unit` and `:integration`.
 
-- search-app
+- [search-app](search-app/README.md)
   - Provides a public search API for concepts in the CMR
   - Main method: `cmr.search.runner`
 
-- search-relevancy-test
+- [search-relevancy-test](search-relevancy-test/README.md)
   - Tests to measure and report the effectiveness of CMR's search relevancy algorithm
 
-- system-int-test
+- [system-int-test](system-int-test/README.md)
   - Black-box, system-level tests to ensure functionality of the CMR
 
-- virtual-product-app
+- [virtual-product-app](virtual-product-app/README.md)
   - Adds the concept of Virtual Products to the CMR. Virtual Products represent
   products that a provider generates on demand from users. This takes place when
    a user places an order or downloads a product through a URL
   - Main method: cmr.virtual_product.runner
 
-- metadata-db-app
+- [metadata-db-app](metadata-db-app/README.md)
   - A database that maintains revisioned copies of metadata for the CMR
   - Main method: cmr.metadata_db.runner
 
-- mock-echo-app
+- [mock-echo-app](mock-echo-app/README.md)
   - This mocks out the ECHO REST API and the URS API as well. Its purpose is to
   make it easier to integration test the CMR system without having to run a full
    instance of ECHO. It will only provide the parts necessary to enable
    integration testing. You should not expect a perfect or complete
    implementation of ECHO.
-  - Main method: cmr.mock_echo.runner
-
-#### Libraries:
-
-- acl-lib
-  - Contains utilities for retrieving and working with ACLs
-
-- common-app-lib
-  - Contains utilities used within many CMR applications
-
-- common-lib
-  - Provides common utility code for CMR projects
-
-- elastic-utils-lib
-  - A library that handles most of the interfacing with Elasticsearch
-
-- es-spatial-plugin
-  - An Elasticsearch plugin that enables spatial search within elastic
-
-- oracle-lib
-  - Contains utilities for connecting to and manipulating data in Oracle
-
-- orbits-lib
-  - Clojure wrapper of a Ruby implementation of the Backtrack Orbit Search
-    Algorithm (BOSA)
-
-- message-queue-lib
-  - A library for interfacing with RabbitMQ, AWS SQS, and an in-memory message queue
-
-- spatial-lib
-  - The spatial libraries provide utilities for working with spatial areas in the CMR
-
-- transmit-lib
-  - The Transmit Library defines functions for invoking CMR services
-
-- umm-lib
-  - This is the old source of UMM schemas and translation code. Since the
-    advent of umm-spec-lib we are planning to remove it
-
-- umm-spec-lib
-  - The UMM Spec lib contains JSON schemas that define the Unified Metadata
-    Model, as well as mappings to other supported formats, and code to migrate
-    collections between any supported formats.
-
-## Further Reading
-
-- [CMR Client Partner User Guide](https://wiki.earthdata.nasa.gov/display/ED/CMR+Client+Partner+User+Guide)
-- [CMR Data Partner User Guide](https://wiki.earthdata.nasa.gov/display/CMR/CMR+Data+Partner+User+Guide)
-
-## License
```

Comment:
should we not continue to leave this at the bottom like everything else on the web?